lib/gitarro/git_op.rb

```diff
@@ -1,131 +1,106 @@
 #! /usr/bin/ruby
```

Comment:
The whole lib/gitarro/git_op.rb was created from scratch, so it is best to view the whole file instead of the diff.